scripts/vm_run_tests.psm1

```diff
@@ -448,8 +598,8 @@ function Invoke-ConnectRedirectTestsOnVM
         }
     }
 
-    $BSTR = [System.Runtime.InteropServices.Marshal]::SecureStringToBSTR($StandardUserPassword)
-    $UnsecurePassword = [System.Runtime.InteropServices.Marshal]::PtrToStringAuto($BSTR)
+    $BSTR = [System.Runtime.InteropServices.Marshal]::SecureStringToBSTR($($StandardUserPassword))
```

Comment:
nit, why is this change needed?

Reply:
This seems more of a gh formatting issue? I only renamed the `$UnsecurePassword` to `$InsecurePassword`.